cmake/Modules/FindRTIConnextDDS.cmake

```diff
@@ -1555,7 +1561,7 @@ if(distributed_logger IN_LIST RTIConnextDDS_FIND_COMPONENTS
     # Find all flavors of rtidlcpp
     set(distributed_logger_cpp_libs
         "rtidlcpp"
-        "librtidlc"
+        "rtidlc"
```

Comment:
i guess this was failing in windows

Reply:
I think this was not even working. But I think it never was used.